common/views/components/GifVideo/GifVideo.js

```diff
@@ -49,6 +49,8 @@ const GifVideo = ({
   const videoRef = useRef(null);
   const canPlayRef = useRef();
   canPlayRef.current = canPlay;
+  const autoPlayDisabledRef = useRef();
```

Comment:
Could you explain a little more about what this is doing, I can't say I can fathonm it.